guidelines/terms/21/motion-animation.html

```diff
@@ -1,7 +1,5 @@
 <dt><dfn id="dfn-motion-animation">motion animation</dfn></dt>
 <dd>
-   					
    <p>addition of steps between conditions to create the illusion of movement or to give a sense of a smooth transition</p>
-	<aside class="example"><p>For example, an element which moves into place or changes size while appearing is considered to be animated. An element which appears instantly without transitioning is not using animation. Motion animation does not include changes of color, blurring, or opacity which do not change the perceived size, shape, or position of the element.</p></aside>
-   				
+	<aside class="example"><p>For example, an element which moves into place or changes size while appearing is considered to be animated. An element which appears instantly without transitioning is not using animation. Motion animation does not include other changes – such as changes of color or opacity – which do not alter the perceived size, shape, position, or distance/depth of the element.</p></aside>
```

Comment:
I suggest the following:
```suggestion
	<aside class="example"><p>For example, an element which moves into place or changes size while appearing is considered to be animated. An element which changes instantly without transition is not using animation. Motion animation does not include other changes even with transition – such as changes of color or opacity – which do not alter the perceived size, shape, position, or distance/depth of the element.</p></aside>
```

Reply:
I'd actually propose a further tweak/change

```suggestion
	<aside class="example"><p>For example, an element which moves into place or changes size while appearing is considered to be animated. An element which appears or changes instantly, without transition/animation steps, is not using animation.</p><p>Motion animation does not include changes – such as changes of color or opacity – which do not alter the perceived size, shape, position, or distance/depth of the element.</p></aside>
```